pyprep/find_noisy_channels.py

```diff
@@ -525,7 +583,9 @@ def run_ransac(
         # Before running, make sure we have enough memory
         try:
             available_gb = virtual_memory().available * 1e-9
-            needed_gb = (data.nbytes * 1e-9) * n_samples
+            needed_gb = (
+                data[: chn_pos.shape[0], :].nbytes * 1e-9
+            ) * n_samples  # chn_pos.shape[0] is just n_chans
```

Comment:
```suggestion
            needed_gb = (
                data[: n_chans, :].nbytes * 1e-9
            ) * n_samples
```

If you had to insert a comment like that, that's a "code smell". (Not your fault, you just put the finger on it)

I suggest to put a line above the "try:"

```Python
n_chans = chn_pos.shape[0]
```

We need `n_chans` further down below anyhow - so this is a better point to introduce it.

Reply:
yeah, I was thinking the same thing when I did it